sql/src/main/java/com/vmware/ddlog/translator/ExpressionTranslationVisitor.java

```diff
@@ -272,6 +277,15 @@ public DDlogExpression process(Node node, @Nullable TranslationContext context)
         return checkHandled(translated, node);
     }
 
+    @Override
+    protected DDlogExpression visitSubqueryExpression(SubqueryExpression node, TranslationContext context) {
+        DDlogIRNode query = context.clone().translateQuery(node.getQuery());
+        RelationRHS rhs = query.to(RelationRHS.class);
+        String relName = context.freshRelationName("sub");
```

Comment:
Is the sub-query always named sub? What happens to multiple sub-queries within one query (e.g., `IN (sub1) AND IN (sub2) ....`)? Or sub-queries across different views?

Reply:
These methods always generated fresh identifiers that do not occur elsewhere.

Reply:
I will add such a test anyway